Makefile

```diff
@@ -0,0 +1,27 @@
+
+SHELL := /bin/bash
+
+.SILENT: clean
+.IGNORE: clean
+
+build:	clean
+	python -m build
+
+clean:
+	rm -rf build/
+	rm -rf dist/
+	rm -rf kafe2.egg-info/
+	rm -rf venv/
+	rm -rf `find . -type d -name __pycache__`
```

Comment:
This is not necessary since the target `clean` is listed in the `.IGNORE` Rule, see [https://www.gnu.org/software/make/manual/html_node/Special-Targets.html](https://www.gnu.org/software/make/manual/html_node/Special-Targets.html) for further information